### PR TITLE
Admin main page phase 1

### PR DESCRIPTION
**Pull request by derekdapp**
Adds a static admin page and home component now redirects user to appropriate dashboard.

### CODE REVIEW

**Comment by TaylorWu21** [comment]
Fix conflicts in the navbar before I merge it

File: client/src/components/Admin/WeeklyHours.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          </Grid.Row>
+        </Grid>
+      </Container>
+    );
+  }
+}
+
+export default withRouter(connect()(WeeklyHours));
+
+// <Divider hidden />
```
Comment (willamzv) [inline client/src/components/Admin/WeeklyHours.js]: if you are not going to use this. just delete

File: client/src/components/Admin/WeeklyHours.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import { withRouter } from 'react-router-dom';
+import { Button,
```
Comment (willamzv) [inline client/src/components/Admin/WeeklyHours.js]: put this in the next line, so everything bellow could have the same indentation

File: client/src/components/NavBar.js [hunk 3]
```diff
@@ -7,15 +7,35 @@ import { withRouter } from 'react-router-dom';
...
+      case 'developer': {
+        return(
+          <Link to={`/timesheets/${moment().format('YYYY/MM/DD')}`}>
+            <Menu.Item name='Dashboard' />
+          </Link>
+        );
+      }
+      default: {
+        return null;
+        break;
```
Comment (willamzv) [inline client/src/components/NavBar.js]: you don't need a break since you already return null

**Comment by derekdapp** [comment]
Since the original pull request, admins can see developer's hours by week on the dash.

**Pull request closed without merging**